.github/workflows/qa.yaml

```diff
@@ -11,19 +11,18 @@ jobs:
   quality:
     name: Code quality and mod files up to date
     runs-on: ubuntu-latest
-    container: ubuntu:latest
```

Comment:
I’m more biased on this one: we want to check our CI against latest release of ubuntu. ubuntu:latest is not ubuntu-latest. It’s the last stable released version of ubuntu. So, I would keep it.